### PR TITLE
[ticket/15492] Fix permission role combobox in RTL

### PR DESCRIPTION
**Pull request by hubaishan**
Varius fixes to css and js for correct
displaying permission role combbox in RTL mode

PHPBB3-15435
PHPBB3-15492

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket:

https://tracker.phpbb.com/browse/PHPBB3-15435
https://tracker.phpbb.com/browse/PHPBB3-15492

### CODE REVIEW

**Comment by hubaishan** [comment]
!set Styles

File: phpBB/adm/style/tooltip.js [hunk 1]
```diff
@@ -130,10 +130,17 @@ phpbb.positionTooltip = function ($element) {
...
-		top: offset.top + 30,
-		left: offset.left - 205
-	});
+	if ($('body').hasClass('rtl')) {
+		$('#_tooltip_container').css({
+			top: offset.top + 30,
+			left: offset.left + 255
+		});
+	} else {
+		$('#_tooltip_container').css({
```
Comment (hanakin) [inline phpBB/adm/style/tooltip.js]: is this an actual id? 🤣

Comment (hubaishan) [reply phpBB/adm/style/tooltip.js]: yes

Comment (hanakin) [reply phpBB/adm/style/tooltip.js]: it will help to review you prs if you provide before and after screenshots

**Comment by hanakin** [comment]
it will help to review you prs if you provide before and after screenshots

**Comment by hubaishan** [comment]
before

**Comment by hubaishan** [comment]
after

**Comment by hubaishan** [comment]
Explain the PR

**Pull request merged**